catalog/transformer/pom.xml

```diff
@@ -21,6 +21,11 @@
     <artifactId>transformer</artifactId>
     <packaging>pom</packaging>
     <name>DDF Catalog Transformers</name>
+
+    <properties>
+        <org.geotools.version>9.5</org.geotools.version>
```

Comment:
✏️ Since this is only used by a single transformer, you might consider moving this to that project.

Reply:
It's used in two of the sub modules, would we rather duplicate it?

Reply:
I guess this is just for the bug fix branch and not master so this should be fine.